vignettes/epidist.Rmd

```diff
@@ -297,38 +299,57 @@ data.frame(
   tab_caption("(ref:pars)")
 ```
 
+For example, we can use the built in `summary()` function
+
+```{r}
```

Comment:
I added this originally to sense check the changes but I also quite like showing this early